src/test/codegen/function-arguments.rs

```diff
@@ -139,6 +139,12 @@ pub fn return_slice(x: &[u16]) -> &[u16] {
   x
 }
 
+// CHECK: i64 @enum_id(i64)
```

Comment:
Isn't this pre-existing? You'd have to take in two arguments and return `{ i32, i32 }` for the `ScalarPair` to be there.

Reply:
I see. I change it to `Result<i64, i64>`.